main/main.c

```diff
@@ -75,6 +77,7 @@ static int parse_args(int argc, char **argv) {
 			break;
 		case 't':
 			args.test_mode = true;
+			packet_trace_enabled = true;
```

Comment:
We may want to enable packet tracing with a real pci device, so test mode and packet tracing should be 2 independent options.

Reply:
Good point, I added `-x`, `--trace-packets`.